Mathlib/Algebra/Algebra/Subalgebra/Basic.lean

```diff
@@ -838,11 +818,11 @@ theorem toSubring_eq_top {R A : Type _} [CommRing R] [Ring A] [Algebra R A] {S :
 #align algebra.to_subring_eq_top Algebra.toSubring_eq_top
 
 theorem mem_sup_left {S T : Subalgebra R A} : ∀ {x : A}, x ∈ S → x ∈ S ⊔ T :=
-  show S ≤ S ⊔ T from le_sup_left
+  have : S ≤ S ⊔ T := le_sup_left; (this ·)
```

Comment:
Possibly worthy of a porting note; the lean4 behavior feels weird here.

Reply:
For what it's worth, I've seen this come up in other places.

Reply:
Added porting note.